### PR TITLE
[DVO-2622] Update aws sdk to v2

### PR DESCRIPTION
**Pull request by aban-ownzones**

### CODE REVIEW

File: src/main/resources/log4j.properties [hunk 1]
```diff
@@ -15,3 +15,13 @@ log4j.appender.CONSOLE.layout=org.apache.log4j.PatternLayout
...
 log4j.appender.CONSOLE.layout.conversionPattern=%d{dd MMM yyyy HH:mm:ss,SSS} [%-5p] [%C{1}]: %m%n
 
+log4j.appender.STDERR=org.apache.log4j.ConsoleAppender
+log4j.appender.STDERR.layout=org.apache.log4j.PatternLayout
+log4j.appender.STDERR.layout.conversionPattern=%d{dd MMM yyyy HH:mm:ss,SSS} [%-5p] [%C{1}]: %m%n
+log4j.appender.STDERR.Threshold=WARN
+log4j.appender.STDERR.Target=System.err
+
+
+#log4j.logger.software.amazon.awssdk=DEBUG, CONSOLE
```
Comment (AdrianMuntean) [inline src/main/resources/log4j.properties:25]: Are these comments needed?

**Review by use-strict — APPROVED** [review]

**Comment by AndreiNet** [comment]
Personally I am familiar with java only at a basic level but I see a lot of changes, some of which I don't know if they are necessary (like changed imports, removed methods' access modifiers and others). Also they are not isolated. I'm concerned that we will run into issues if we will want to bring Photon up to date with the original repo.

**Comment by aban-ownzones** [comment]
> 
it's just Idea formatting - depends on your settings, but they're fine.
Removing 'public' access modifier from public interfaces is fine, as they're already public.

File: src/main/resources/log4j.properties (hunk 1 above)
Comment (aban-ownzones) [reply src/main/resources/log4j.properties:25]: yup yup, for the next peasant that has to debug it, they can uncomment

**Review by mvinturis — APPROVED** [review]

File: build.gradle [hunk 2]
```diff
@@ -77,6 +77,10 @@ dependencies {
     compileOnly "com.github.spotbugs:spotbugs-annotations:${spotbugs.toolVersion.get()}"
     implementation "org.slf4j:slf4j-api:1.7.+"
     implementation "org.slf4j:slf4j-log4j12:1.7.+"
```
Comment (andreibalan) [inline build.gradle:79]: We should create a separate task in order to upgrade this to the latest 2.x version of the log4j library in order to remove a bunch of critical, high, and medium CVE vulnerabilities

Comment (aban-ownzones) [reply build.gradle:79]: https://github.com/OwnZones/photon/pull/12

**Pull request merged**